dota2/msg.py

```diff
@@ -110,4 +110,7 @@ def find_proto(emsg):
     EDOTAGCMsg.EMsgClientToGCSetPartyLeader: dota_gcmessages_client_match_management_pb2.CMsgDOTASetGroupLeader,
     EDOTAGCMsg.EMsgGCOtherJoinedChannel: dota_gcmessages_client_chat_pb2.CMsgDOTAOtherJoinedChatChannel,
     EDOTAGCMsg.EMsgGCOtherLeftChannel: dota_gcmessages_client_chat_pb2.CMsgDOTAOtherLeftChatChannel,
+    EDOTAGCMsg.EMsgGCPracticeLobbyResponse: base_gcmessages_pb2.CMsgLobbyInviteResponse,
+    EDOTAGCMsg.EMsgGCPracticeLobbyKick: dota_gcmessages_client_match_management_pb2.CMsgPracticeLobbyKick,
+    EDOTAGCMsg.EMsgGCPracticeLobbyKickFromTeam: dota_gcmessages_client_match_management_pb2.CMsgPracticeLobbyKickFromTeam,
 }
```

Comment:
Does the function not find these `EMsgGCPracticeLobbyKick` and `EMsgGCPracticeLobbyKickFromTeam`?